### PR TITLE
Adding possibility to add short descriptions for sessions.

### PR DESCRIPTION
**Pull request by bstrandjev**
This is especially useful for events with only few tracks. In such case users do not think of clicking any session to see the details.

session-description

### CODE REVIEW

File: src/elements/session-element.html [hunk 1]
```diff
@@ -149,6 +153,7 @@
       <div class="session-header" layout horizontal justified>
         <div flex>
           <h3 class="session-title">[[session.title]]</h3>
+          <div class="session-description" hidden$="[[!session.shortDescription]]">[[session.shortDescription]]</div>
```
Comment (ozasadnyy) [inline src/elements/session-element.html]: Could you make it optional via `settings.json` https://github.com/gdg-x/hoverboard/blob/master/data/settings.json#L19  Add some property as `enableShortDescription`and basing on it render or not render the current block

Comment (bstrandjev) [reply src/elements/session-element.html]: @ozasadnyy  I definitely approve the idea of feature flags, but the case is that the field will be optional even if the option is enabled (it is not necessary to provide description for each session). Thus, I wonder if the feature flag is needed at all - we just omit the element if the property does not exist. What do you think?

Comment (ozasadnyy) [reply src/elements/session-element.html]: I would rather have this flag in a case when organizers want to postpone publishing of description until they fill all of them or any other cases

Comment (bstrandjev) [reply src/elements/session-element.html]: @ozasadnyy I have read what you have said and it makes sense. For the moment I am too occupied at work, will try to improve the PR in the following days

**Pull request merged**